clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractFetch.java

```diff
@@ -330,7 +343,7 @@ private Map<Node, FetchSessionHandler.FetchRequestData> prepareCloseFetchSession
                 // skip sending the close request.
                 final Node fetchTarget = cluster.nodeById(fetchTargetNodeId);
 
-                if (fetchTarget == null || client.isUnavailable(fetchTarget)) {
+                if (fetchTarget == null || isUnavailable(fetchTarget)) {
```

Comment:
Per previous comment - if we actually clean up the code, we should only care if there's a fetchTarget or not.

Reply:
True. We'll handle that in KAFKA-15556.